app/services/invite_schools.rb

```diff
@@ -97,6 +97,32 @@ def invite_to_beta(school_urns)
     end
   end
 
+  def invite_mats(school_urns)
+    school_urns.each do |urn|
+      school = School.eligible.find_by(urn: urn)
+
+      if school.nil?
+        logger.info "School not found, urn: #{urn} ... skipping"
+        next
+      end
+
+      if school.contact_email.blank?
+        logger.info "No contact details for school urn: #{urn} ... skipping"
+        next
+      end
```

Comment:
Feels like this logic gets repeated a lot in this file. Should it be extracted to something like `find_school_to_email` method that can return `nil`, and log some relevant info?

Reply:
Agreed. I'm going to put this in for now, then take a look